world.py

```diff
@@ -120,3 +123,28 @@ def next_level(self, level):
         level += 1
         if os.path.isfile('data/map/%d.txt' % level):
             return level
+    
+    def init_surfaces(self):
+        paths = {
+            "intro-1": "intro/1.jpg",
+            "intro-2": "intro/2.jpg",
+            "intro-3": "intro/3.jpg",
+            "intro-4": "intro/4.jpg",
+            "intro-5": "intro/5.jpg",
+            "background": "background.jpg",
+            "title": "title_background.jpg",
+            "end": "end/festejo.jpg",
+        }
+
+        self.surfaces = {}
+        screen_w, screen_h = self.screen.get_size()
+
+        # intro images
```

Comment:
Comment is incorrect.

Reply:
wrote this when I was thinking some other fix. forgot to remove afterward. removing.